CHANGELOG.md

```diff
@@ -16,6 +16,10 @@
   [Jordan Rose](https://github.com/jrose-signal)
   [SimplyDanny](https://github.com/SimplyDanny)
 
+* Add `Cognitive Complexity Rule`
```

Comment:
If you look at previous rule addition changelog entries, they tend to be a bit wordier, and to specify the rule by its identifier. e.g.

```
* Add new `contrasted_opening_brace` rule that enforces opening
  braces to be on a separate line after the preceding declaration.  
  [SimplyDanny](https://github.com/SimplyDanny)
```